tl.tl

```diff
@@ -2217,7 +2218,7 @@ local function parse_variable_declarations(ps: ParseState, i: number, node_name:
    if #asgn.vars == 0 then
       return fail(ps, i, "expected a local variable definition")
    end
-   local lhs: Node = asgn.vars[1]
+   -- local lhs: Node = asgn.vars[1]
```

Comment:
I think this is a left over from when type declarations were done with plain `local` and can go away too